public/main.js

```diff
@@ -1782,6 +1792,124 @@ const containerComp = Vue.component('wands-container', {
     </div>`,
 })
 
+const pillarsComp = Vue.component('pillars-comp', {
+    data() {
+        return {
+            apothState: false,
+        }
+    },
+    computed: {
+        info() {
+            const pillarVersion = this.apothState ? pillarsApoth : pillars
+            const lengths = pillarVersion.map((x) => x.length - 1)
+            return {
+                pillarVersion,
+                lengths,
+                headers: pillarVersion.map((x) => x[0].name),
+                spacing: lengths.map((x) => Math.max(...lengths) - x),
+                maxLengths: Math.max(...pillarVersion.map((x) => x.length - 1)),
+            }
+        },
```

Comment:
This could be DRY'd up a bit:

```suggestion
        info() {
            const pillarVersion = this.apothState ? pillarsApoth : pillars
            const lengths = pillarVersion.map((x) => x.length - 1)
            const maxLengths = Math.max(...lengths);
            return {
                pillarVersion,
                lengths,
                headers: pillarVersion.map((x) => x[0].name),
                spacing: lengths.map((x) => maxLengths - x),
                maxLengths,
            }
        },
```

Reply:
(note: I think if you go with the code below, the lengths and stuff here are perhaps not necessary?)

Reply:
After looking over this, maxLengths is straight up not needed, in a previous implementation of this I used it but I cut it out.  

Looking it over, I *should* be able remove lengths from the return object as it is only used in the progress() computed and I'll be rewriting it to something similar to what you suggested below.

pillarVersion, headers, and spacing are all used in the template HTML rendering so they remain necessary.